tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_KerrSchild.cpp

```diff
@@ -366,13 +380,116 @@ void test_zero_spin_optimization(const DataType& used_for_size) {
       });
 }
 
+template <typename Frame, typename DataType>
+void test_boosted_for_zero_velocity(const DataType& used_for_size) {
+  const std::array<double, 3> boost_velocity{{0.0, 0.0, 1.0e-50}};
+
+  gr::Solutions::KerrSchild solution_zero_velocity(
+      3.0, {{0., 0., 0.}}, {{0.2, 0.3, 0.2}});
+  gr::Solutions::KerrSchild solution_tiny_velocity(
+      3.0, {{0., 0., 1e-50}}, {{0.2, 0.3, 0.2}}, boost_velocity);
+  CHECK(solution_zero_velocity.zero_velocity());
+  CHECK(not solution_tiny_velocity.zero_velocity());
+  const auto x = spatial_coords<Frame>(used_for_size);
+  using all_tags = typename gr::Solutions::KerrSchild::tags<DataType, Frame>;
+  const auto all_tags_zero_velocity =
+      solution_zero_velocity.variables(x, 0., all_tags{});
+  const auto all_tags_tiny_velocity =
+      solution_tiny_velocity.variables(x, 0., all_tags{});
+  tmpl::for_each<all_tags>(
+      [&all_tags_zero_velocity, &all_tags_tiny_velocity](auto tag_v) {
+        using tag = tmpl::type_from<decltype(tag_v)>;
+        CHECK_ITERABLE_APPROX(get<tag>(all_tags_zero_velocity),
+                              get<tag>(all_tags_tiny_velocity));
+      });
+}
+
+template <typename Frame, typename DataType>
+void test_boosted_spacetime_metric(const DataType& used_for_size) {
+  // Test that the extrinsic curvature of the boosted solution indeed
+  // corresponds to the boosted extrinsic curvature tensor
+
+  const auto x = spatial_coords<Frame>(used_for_size);
+  const double t = 0.0;
+
+  // KerrSchild solution
+  const double mass = 1.0;
+  const std::array<double, 3> spin{{0.0, 0.0, 0.0}};
+  const std::array<double, 3> center{{0.0, 0.0, 0.0}};
+  gr::Solutions::KerrSchild solution(mass, spin, center);
+
+  // Boosted KerrSchild solution
+  const std::array<double, 3> velocity{{0.5, 0.0, 0.0}};
+  gr::Solutions::KerrSchild boosted_solution(mass, spin, center, velocity);
+
+  // Need to evaluate at the boosted coordinates
+  // Boost coordinates
```

Comment:
These are the unboosted coords, no? (related to comments above)

Reply:
Maybe the terminology ''boosted/unboosted'' is a bit confusing.  Rest frame (of the BH) coords I would say.